rs/tests/nns/BUILD.bazel

```diff
@@ -16,7 +16,7 @@ system_test(
         "vcpus": 16,
         "boot_image_minimal_size_gibibytes": 100,
     },
-    env = NNS_CANISTER_ENV,
+    env = IC_MAINNET_NNS_RECOVERY_ENV,
```

Comment:
Wdyt, do we want mainnet or tip-of-master here, or both? 

Cc @max-dfinity @daniel-wong-dfinity-org

Reply:
Just double checking

Reply:
For example, FI tests use this:

```
    # Run this test using the mainnet version of the pocket-ic-server
    # but only run this test using the HEAD version of the pocket-ic server nightly.
    extra_HEAD_tags = ["fi_tests_nightly"],
```

Reply:
@aterga note that, as seen above, `NNS_CANISTER_ENV` is still included in `IC_MAINNET_NNS_RECOVERY_ENV` so this system-test is still using the HEAD NNS canisters. The only thing that changed here is that the environment now also contains environment variables that point to the mainnet ic-replay and ic-recovery.